src/main/scala/mesosphere/marathon/core/instance/Instance.scala

```diff
@@ -132,5 +134,11 @@ object Instance {
   implicit val idFormat = Json.format[Instance.Id]
   implicit val instanceStatusFormat = Json.format[InstanceStatus]
   implicit val instanceStateFormat = Json.format[InstanceState]
-  implicit val instanceJsonFormat = Json.format[Instance]
+  implicit val instanceJsonFormat: Format[Instance] = Json.format[Instance]
+
+  import mesosphere.mesos.Placed
+  implicit class PlacedInstance(instance: Instance) extends Placed {
```

Comment:
I don't even fully understand what you're doing. I haven't used view bounds since 2.8, so this is just funky.